### DISCUSSION TITLE
what are exactly the anti-features listed on F-Droid?

### DISCUSSION THREAD

**Discussion by Nekothegamer** [deckerst/aves#463] [Q&A] [answered]
On F-Droid these anti-features are listed:

• Has non-free dependencies
• Promotes non-free network services

what are those non free dependencies? what are the non free network services? this is important, it help giving the app development more transparency and because of this I'm probably not the only who asked for this.

**Comment by deckerst** [comment, marked as answer]
You can see the [wiki](https://github.com/deckerst/aves/wiki/App-Versions) for the version differences. As for what is deemed an anti-feature, i believe it's the display of Google Maps in the izzy version.
